src/server/app.js

```diff
@@ -28,14 +33,85 @@ if (process.env.NODE_ENV === 'production-notugly') {
   )
 }
 
-const staticHandler = express.static(path.resolve(__dirname, '../../', 'build'))
+const hbs = exphbs.create({
+  // Specify helpers which are only registered on this instance.
+  extname: '.hbs',
+  helpers: {
+    foo: function () {
+      return 'FOO!'
+    },
+    bar: function () {
```

Comment:
Can we still put each route in a separate file so that this module does not get too big and complex?

Reply:
ok